src/themes/dspace/app/+home-page/home-news/home-news.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="background-image">
+  <div class="container">
+    <div class="jumbotron jumbotron-fluid">
+      <div class="d-flex flex-wrap">
+        <div>
+          <h1 class="display-3">DSpace 7</h1>
+          <p class="lead">DSpace is the world leading open source repository platform that enables
+            organisations to:</p>
+        </div>
+      </div>
+      <ul>
+        <li>easily ingest documents, audio, video, datasets and their corresponding Dublin Core
+          metadata
+        </li>
+        <li>open up this content to local and global audiences, thanks to the OAI-PMH interface and
+          Google Scholar optimizations
+        </li>
+        <li>issue permanent urls and trustworthy identifiers, including optional integrations with
+          handle.net and DataCite DOI
+        </li>
+      </ul>
+      <p>Join an international community of <a href="https://wiki.duraspace.org/display/DSPACE/DSpace+Positioning" target="_blank">leading institutions using DSpace</a>.</p>
```

Comment:
Tiny thing, this link should be updated to the new wiki URL (i.e. lyrasis.org): https://wiki.lyrasis.org/display/DSPACE/DSpace+Positioning

Reply:
Fixed